### ISSUE TITLE
Add to foodsharing docker setup

### ISSUE THREAD

**Issue by nicksellen** [foodsharing-dev/foodsharing-api#6]
Should add it to foodsharing docker setup, should be able to use same PHP image. As it is a different src repo I would suggest mounting a subdirectory containing a manual git clone of this repo, then add a volume for `vendor` dir (so composer deps stay inside the container), and just manually run composer commands inside the container for now....

Maybe @raph-ael you want to have a go with this? Might be a good way to get into this docker stuff...

**Comment by NerdyProjects** [comment]
@raph-ael :

> mybe its possible to make a new docker for new frontend and backend and put the old backendfrontentchatsocketdocker inside the new docker.
> docker homepage said its possible: https://blog.docker.com/2013/09/docker-can-now-run-within-docker/

**Comment by NerdyProjects** [comment]
I would not do that. Because we actually just need a small change in the webserver config as well as another path binding for the new app in the existing docker configuration. Creating the whole new setup "outside" sounds more complicated and I don't see any reason. But feel free to explain why it should be an advantage

**Comment by nicksellen** [comment]
Yes I think that would get very complicated very quickly. Much easier to add another container in the existing one, just copy the definition for the app one to start with.
